tracker/models.py

```diff
@@ -0,0 +1,66 @@
+from django.db import models
+
+
+# Create your models here.
+class User(models.Model):
+    name = models.CharField(max_length=256)
+    last_Name = models.CharField(max_length=256)
+    login = models.CharField(max_length=256, unique=True)
+    password = models.CharField(max_length=256)
+
+
+class Customer(User):
+    age = models.PositiveIntegerField()
+    height = models.FloatField()
+    goal = models.CharField(max_length=256)
+
+
+class Admin(User):
+    email = models.EmailField(unique=True)
+
+
+class Exercise(models.Model):
+    name = models.CharField(max_length=256)
+    type = models.CharField(max_length=256)
+    description = models.TextField()
+
+
+class WeightTracker(models.Model):
+    user = models.ForeignKey("User", on_delete=models.CASCADE)
```

Comment:
To use this program, does the administrator need to have another account, like user? Why?